### DISCUSSION TITLE
Inbox rate

### DISCUSSION THREAD

**Discussion by nagireddybi** [postalserver/postal#2015] [Feature suggestions]
Hello Team

Can we add the total inbox rate in postal like how many emails landed in the inbox?

**Comment by willpower232** [comment]
It wouldn't be very accurate but you could count the number of messages in the database I guess. Postal isn't really intended for long term storage of messages so you might be better off integrating webhooks and handling this information in the system that sends the messages via Postal.

**Reply by nagireddybi** [reply]
Thank you, I am aware that the postal stores opens and clicks. Can we assess the inbox rate mean how many emails landed in the inbox rather than the spam based on the opens or clicks. Is there any logic we can implement to assess the inbox rate of postal or its IPs. Thank you. accept my apologies if I am not clear.

**Reply by willpower232** [reply]
IMHO the only way to know for sure that an email arrived in the inbox is via the opens and clicks. Even if the response from the mail server was that it was received, they may still junk it before the user gets anywhere near it.

You would have to check the database directly or implement Postals webhooks to create the statistics yourself

**Reply by nagireddybi** [reply]
Thank you for guidance on this subject.
